core/ui/src/views/SoftwareCenterCoreApps.vue

```diff
@@ -0,0 +1,449 @@
+      this.error.listCoreModules = "";
+      this.loading.listCoreModules = true;
+      const taskAction = "list-core-modules";
+      const eventId = this.getUuid();
+
+      // register to task error
+      this.$root.$once(
+        `${taskAction}-aborted-${eventId}`,
+        this.listCoreModulesAborted
+      );
+
+      // register to task completion
+      this.$root.$once(
+        `${taskAction}-completed-${eventId}`,
+        this.listCoreModulesCompleted
+      );
+
+      const res = await to(
+        this.createClusterTask({
+          action: taskAction,
+          extra: {
+            title: this.$t("action." + taskAction),
+            isNotificationHidden: true,
+            eventId,
+          },
+        })
+      );
+      const err = res[0];
+
+      if (err) {
+        console.error(`error creating task ${taskAction}`, err);
+        this.error.listCoreModules = this.getErrorMessage(err);
+        this.loading.listCoreModules = false;
+        return;
+      }
+    },
+    listCoreModulesAborted(taskResult, taskContext) {
+      console.error(`${taskContext.action} aborted`, taskResult);
+      this.error.listCoreModules = this.$t("error.generic_error");
+      this.loading.listCoreModules = false;
+    },
+    listCoreModulesCompleted(taskContext, taskResult) {
+      let coreModules = taskResult.output;
+      // update node_id with this.getNodeLabel() to  implement the serch by node_id and node_ui_name
+      coreModules = coreModules.map((coreApp) => {
+        coreApp.instances = coreApp.instances.map((instance) => {
+          instance.node_id = this.getNodeLabel({
+            id: instance.node_id,
+            ui_name: instance.node_ui_name,
+          });
+          return instance;
+        });
+        return coreApp;
+      });
+
+      let isCoreUpdateAvailable = false;
+
+      this.coreApp = {
+        id: "core",
+        name: this.$t("software_center.core_app_name", {
+          productName: this.$root.config.PRODUCT_NAME,
+        }),
+        description: {
+          en: this.$t("software_center.core_app_description", {
+            productName: this.$root.config.PRODUCT_NAME,
+          }),
+        },
+        installed: coreModules,
+      };
+
+      // check for core updates
+
+      for (const coreApp of coreModules) {
+        for (const coreInstance of coreApp.instances) {
+          if (coreInstance.update) {
+            isCoreUpdateAvailable = true;
+          }
+        }
+      }
+
+      if (isCoreUpdateAvailable) {
+        this.isCoreUpdateAvailable = true;
+      } else {
+        this.isCoreUpdateAvailable = false;
+      }
+      this.coreModules = coreModules;
+      this.loading.listCoreModules = false;
+    },
+
+    onWillUpdateCore(coreUpdate) {
+      const notification = {
+        id: this.getUuid(),
+        title: this.$t("software_center.core_app_are_going_to_be_updated"),
+        type: "info",
+        toastTimeout: this.DELETE_DELAY - 1000,
```

Comment:
This subtraction look a bit odd, I'd use `this.DELETE_DELAY`  or a simple value, e.g. `6000`